src/js/BookReader.js

```diff
@@ -277,7 +283,11 @@ BookReader.prototype.initParams = function() {
     // params explicitly set in URL take precedence over all other methods
     var urlParams = this.paramsFromFragment(this.urlReadFragment());
     if (urlParams.mode) {
-      this.prevReadMode = urlParams.mode;
```

Comment:
Why was this assignment removed? I didn't see any corresponding changes to how this property is read or interpreted. It seems like it's used when in thumbnail view and a leaf is clicked/tapped, the previous mode is switched to. Does this omission affect that if BookReader starts in thumbnail mode?

Reply:
Thanks for catching this. Don't remember why it was removed
Moved setting this variable to BookReader.prototype.init